CoreScriptsRoot/CoreScripts/NotificationScript2.lua

```diff
@@ -429,37 +521,99 @@ local function onFriendRequestEvent(fromPlayer, toPlayer, event)
 	--
 	if fromPlayer == LocalPlayer then
 		if event == Enum.FriendRequestEvent.Accept then
-			sendNotifcation("New Friend", "You are now friends with "..toPlayer.Name.."!",
-				FRIEND_IMAGE..tostring(toPlayer.userId).."&x=48&y=48", DEFAULT_NOTIFICATION_DURATION, nil)
+			if newNotificationPath then
+				sendNotificationInfo {
+					GroupName = "FriendRequests",
+					Title = "New Friend",
+					Text = toPlayer.Name,
+					DetailText = "You are now friends with " .. toPlayer.Name .. "!",
+					
+
+					Image = getFriendImage(toPlayer.userId),
+					Duration = DEFAULT_NOTIFICATION_DURATION
+				}
+			else
+				sendNotification("New Friend", "You are now friends with "..toPlayer.Name.."!",
+					FRIEND_IMAGE..tostring(toPlayer.userId).."&x=48&y=48", DEFAULT_NOTIFICATION_DURATION, nil, nil, nil, "FriendRequests")
+			end
 		end
 	elseif toPlayer == LocalPlayer then
 		if event == Enum.FriendRequestEvent.Issue then
 			if FriendRequestBlacklist[fromPlayer] then return end
 			sendFriendNotification(fromPlayer)
 		elseif event == Enum.FriendRequestEvent.Accept then
-			sendNotifcation("New Friend", "You are now friends with "..fromPlayer.Name.."!", 
-				FRIEND_IMAGE..tostring(fromPlayer.userId).."&x=48&y=48", DEFAULT_NOTIFICATION_DURATION, nil)
+			if newNotificationPath then
+				sendNotificationInfo {
+					GroupName = "FriendRequests",
+					Title = "New Friend",
+					Text = fromPlayer.Name,
+					DetailText = "You are now friends with " .. fromPlayer.Name .. "!",
+
+					Image = getFriendImage(fromPlayer.userId),
+					Duration = DEFAULT_NOTIFICATION_DURATION
+				}
+			else
+				sendNotification("New Friend", "You are now friends with "..fromPlayer.Name.."!", 
+					FRIEND_IMAGE..tostring(fromPlayer.userId).."&x=48&y=48", DEFAULT_NOTIFICATION_DURATION, nil, nil, nil, "FriendRequests")
+			end
 		end
 	end
 end
 
 --[[ Player Points Notifications ]]--
 local function onPointsAwarded(userId, pointsAwarded, userBalanceInGame, userTotalBalance)
-	if pointsNotificationsActive and userId == LocalPlayer.userId then
-		if pointsAwarded == 1 then
-			sendNotifcation("Point Awarded", "You received "..tostring(pointsAwarded).." point!", PLAYER_POINTS_IMG, DEFAULT_NOTIFICATION_DURATION, nil)
-		elseif pointsAwarded > 0 then
-			sendNotifcation("Points Awarded", "You received "..tostring(pointsAwarded).." points!", PLAYER_POINTS_IMG, DEFAULT_NOTIFICATION_DURATION, nil)
-		elseif pointsAwarded < 0 then
-			sendNotifcation("Points Lost", "You lost "..tostring(-pointsAwarded).." points!", PLAYER_POINTS_IMG, DEFAULT_NOTIFICATION_DURATION, nil)
+	if newNotificationPath then
+		if pointsNotificationsActive and userId == LocalPlayer.userId then
+			local title, text, detailText
+			if pointsAwarded == 1 then
+				title = "Point Awarded"
+				text = "You received 1 point!"
+			elseif pointsAwarded > 0 then
+				title = "Points Awarded"
+				text = ("You received %d points!"):format(pointsAwarded)
+			elseif pointsAwarded < 0 then
```

Comment:
It might be good to handle the 0 point case just in-case something wonky goes on